components/constellation/constellation.rs

```diff
@@ -1446,7 +1446,7 @@ where
         let (source_pipeline_id, content) = message;
         trace_script_msg!(content, "{source_pipeline_id}: {content:?}");
 
-        let source_top_ctx_id = match self
+        let webview_id = match self
```

Comment:
Why is the variable renamed while the type is still `TopLevelBrowsingContextId`?

Reply:
`WebViewId` is an alias for `TopLevelBrowsingContextId`, so they are the same type. In a future change we will remove `TopLevelBrowsingContextId`, but this is still a time of transition.